src/App/App.js

```diff
@@ -12,6 +13,8 @@ class App extends Component {
       movies: [],
       error: '',
 			displayLoginPage: false,
+			displayMovieDetails: false,
```

Comment:
Keep this in mind as a refactoring opportunity tomorrow during the Router lesson.